nginx/datadog_checks/nginx/__init__.py

```diff
@@ -2,9 +2,10 @@
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
 from .__about__ import __version__
-from .nginx import Nginx
+from .nginx import Nginx, VTS_METRIC_MAP
```

Comment:
I assume this is being exported for the test?

Reply:
Yep, if you think it's not worth it I can import the name with the full path, open to suggestions